tests/scripts/test-paths.js

```diff
@@ -0,0 +1,8 @@
+exports.getLocalPath = function(){
```

Comment:
Let's make this work the same way it did in dragoon.  Add an example test path file, git rm this one, and add tests/scripts/test-paths.js to the gitignore.